packages/driver/src/cy/assertions.ts

```diff
@@ -77,7 +81,7 @@ export const create = (Cypress, cy) => {
   const getUpcomingAssertions = () => {
     const index = cy.state('index') + 1
 
-    const assertions = []
+    const assertions: any[] = []
```

Comment:
The specific type of `any` is [`$Command` here](https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/cypress/command.ts). But I made it to `any` for now. Because it should be worked with `$Cy.queue` and the other `$Cy` members.

Reply:
Do we want to have the types be explicit with the the other cy members or address that in another PR? For now `any` seems like it fits the bill 😄 .

Reply:
Right. Removing `// @ts-nocheck` from `pkg/driver` is a big work. So, I'm breaking them down to bite-size. 

(Actually, there are sometimes so many things to think. So I'm postponing them for the later PRs.)